packages/documentation-spa/blog/2021-08-09-state-of-one-platform-jul-2021.md

```diff
@@ -0,0 +1,97 @@
+---
+slug: state-of-one-platform-jul-2021
+title: State of One Platform - Jul 2021
+author: Ghanshyam Lohar
+author_title: Senior Software Engineer @ Red Hat
+author_url: https://github.com/ghanlohar
+author_image_url: https://avatars.githubusercontent.com/u/5575651?s=400&v=4
+tags: [redhat, blog, oneportal, oneplatform, react, angular, vue, jul, update]
+---
+### Meta
+**Completed Sprints:** OP – Jul'21 Mid Sprint, OP – Jul'21 End Sprint
+
+**Ongoing/Upcoming Sprint:** OP – Aug'21 Mid Sprint
+
+**One Platform:** [one.redhat.com](https://one.redhat.com)
+
+**JIRA:** [https://issues.redhat.com/projects/ONEPLAT/issues](https://issues.redhat.com/projects/ONEPLAT/issues)
+
+**GitHub:** [https://github.com/1-Platform/one-platform/issues](https://github.com/1-Platform/one-platform/issues)
+
+### Introduction
+This is a summary of the development goals achieved by the _One Platform team_ in the month of **July** 2021.
+
+_This month, the strategy for the team was changed and the 4 weeks Sprint cycle was reduced to  2 Weeks cycle_.  As part of two Sprints in July’21, One Platform Team had the following high-level goals which have been achieved as part of **113** **Jira** Issues closed by the Team.
+
+* **Application Development**:
+    * _Home Page_
+        * Rebranding the home page as _Application hub._
+        * Contact us page is updated with new team members data.
+    * _Search_
+        * Indexing native microservice data to Solr - Blocked testing on pre prod environments due to whitelisting of preprod environments in Akamai.
```

Comment:
```suggestion
        * Indexing native microservice data to Solr - Testing in pre-production environments is blocked because few configurations needs to be whitelisted with Akamai.
```